g19.go

```diff
@@ -0,0 +1,6 @@
+// +build go1.9
```

Comment:
Is this the most robust solution for unique build code?

I am wondering if we could implement some auto-generating system to solve this problem moving forward? I opened up https://github.com/StabbyCutyou/generics/issues/20 to take the discussion forward

Reply:
This is a good question - i'd invite the opener of this PR into the discussion to get his thoughts as well.

Reply:
The build constraint will allow building for future, at least until the Go Team will see the errors of their way and adopt this excellent generics solution as a solution for generics in Go.